content/13.conclusion.md

```diff
@@ -1,9 +1,13 @@
 ## Conclusion {#conclusion}
 
-Collectively, our manuscript is focused on the promotion of practical tips, cutting-edge insights, and evolving professional standards that can help to advance the efficient and optimal application of deep learning across a wide range of scientific research environments.
-Specifically, we have tried to increase the accessibility of deep learning techniques within the life sciences so that these methods can generate innovative biomedical insights, tools, technologies, services, and products.
-A number of our tips (see Tips 7, 8, 9, and 10) are deeply motivated by the need to safeguard against key risks, which include the introduction and perpetuation of bias, over- or mis-interpretation of models, poor generalizability, and the potential for harm unto others.
-However, despite these concerns, deep learning techniques have the potential to add value across a diverse array of research and healthcare contexts if they are leveraged in ethical and responsible ways.
-After all, these techniques have already shown a remarkable ability to meet or exceed the performance of human effort and older algorithms across numerous fields and subdisciplines.
-In sum, we hope this work will encourage the development of a prudent and vigilant community of computational practitioners, experimental biologists, and clinical scientists that think carefully and critically about their usage and deployment of powerful deep learning techniques.
-After all, the initial discernment of the potential scientific and societal consequences of one's work is critical to the advancement of accountable, beneficent, and high-quality science.
+Collectively, our manuscript is focused on the promotion of practical tips distilled from cutting-edge insights and evolving professional standards to advance the efficient and optimal application of deep learning within research.
```

Comment:
Not sure if *within* is the right word. 

Personally, maybe instead of

> of deep learning within research.

the following 

> of deep learning for biological research.

(or *life science research*)

would sound a bit better?

May be a native speaker could take a look.